src/new/templates.js

```diff
@@ -9,53 +9,56 @@ export function api(apiName)  {
 }`
 }
 
-export function env(env)  {
+export function env (env) {
   return `module.exports = {
   env: "${env}",
   secretkey: "${env}-secret-key"
 }`
 }
 
-export function gitignore()  {
+export function gitignore () {
   return `dist/*
 node_modules/*
-functions/*/node_modules/*
 config/*`
 }
 
-export function lambda(){
+export function lambda () {
   let obj = {
-    Handler: "index.handler",
+    Handler: 'index.handler',
     MemorySize: 128,
-    Role: "",
+    Role: '',
     Timeout: 10,
-    Runtime: "nodejs4.3"
+    Runtime: 'nodejs4.3'
   }
 
   return JSON.stringify(obj, null, 2)
 }
 
-export function pkg(apiName, api)  {
+export function pkg (apiName) {
   let obj = {
     name: apiName,
-    version: "1.0.0",
+    version: '1.0.0',
     private: true,
-    description: "",
-    license: "",
-    dependencies: {},
-    shep: {}
+    description: '',
+    license: '',
+    devDependencies: {
+      webpack: '2.1.0-beta.25'
+    },
+    shep: {
+      region: '',
+      accountId: '',
+      apiId: ''
+    }
   }
 
-  if (api === false) { obj.shep.api = false }
-
   return JSON.stringify(obj, null, 2)
 }
 
-export function readme(apiName){
+export function readme (apiName) {
   return `# ${apiName}`
 }
 
-export function webpack(){
+export function webpack () {
```

Comment:
How do you feel about switching from handling which functions to bundle from env config to directly passing the entry points to command line? i.e. in `src/util/build-functions.js` have `exec('webpack', entryPoints, { env: { NODE_ENV } })` where entryPoints is the entry object from the above `webpack.config.js`. IMO this would make the default webpack config more straightforward.